cms/static/js/views/course_info_handout.js

```diff
@@ -50,10 +50,14 @@ define(['js/views/baseview', 'codemirror', 'common/js/components/views/feedback_
             },
 
             onSave: function(event) {
+                var dataToBeSaved = this.$codeMirror.getValue();
                 $('#handout_error').removeClass('is-shown');
                 $('.save-button').removeClass('is-disabled').attr('aria-disabled', false);
                 if ($('.CodeMirror-lines').find('.cm-error').length == 0) {
-                    this.model.set('data', this.$codeMirror.getValue());
+                    if (dataToBeSaved === '') {
```

Comment:
That's fine as long as saving blank lines/spaces is respected (ie, the initial bug is not triggered). I wasn't sure what the behavior would be in that case.